### PR TITLE
[WFLY-12823] Quickstart for MP REST-client 1.4

### PR DESCRIPTION
**Pull request by xstefank**
https://issues.jboss.org/browse/WFLY-12823

### CODE REVIEW

File: microprofile/WFLY-12823_EAP7-1375_quickstart-for-mp-rest-client-1.3.adoc [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+=== Affected Projects or Components
+
+* Quickstarts
+
+=== Other Interested Projects
+
+== Requirements
+
+=== Hard Requirements
```
Comment (honza-kasik) [inline microprofile/WFLY-12823_EAP7-1375_quickstart-for-mp-rest-client-1.3.adoc]: Please describe the steps of quickstart which will user go through. These key points are IMHO not sufficient for analysis document.

File: microprofile/WFLY-12823_EAP7-1375_quickstart-for-mp-rest-client-1.3.adoc [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+* definition of the REST client interface
+* providers (@RegisterProvider, ResponseExceptionMapper)
+* async support
+* CDI lookup (@RegisterRestClient, @RestClient)
+* programmatic lookup (RestClientBuilder)
+* configurable MP Config values
+
+=== Non-Requirements
+
+* AsyncInvocationInterceptors
```
Comment (honza-kasik) [inline microprofile/WFLY-12823_EAP7-1375_quickstart-for-mp-rest-client-1.3.adoc]: What does this mean?

**Comment by xstefank** [comment]
@honza-kasik can you take a look again, please?

File: microprofile/WFLY-12823_EAP7-1375_quickstart-for-mp-rest-client-1.3.adoc [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+* Quickstarts
+
+=== Other Interested Projects
+
+== Requirements
+
+=== Hard Requirements
+
+* individual steps that the users will go through:
+** create a model JSON class for the external site
```
Comment (honza-kasik) [inline microprofile/WFLY-12823_EAP7-1375_quickstart-for-mp-rest-client-1.3.adoc]: Point discussed off GitHub. The dependency on external site is not desired here...

**Pull request merged**